Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -305,20 +322,31 @@ void IBocColumnRenderer.RenderDataCell (BocColumnRenderingContext renderingConte
     /// This is a template method. Deriving classes must implement <see cref="RenderCellContents"/> to provide the contents of
     /// the table cell (&lt;td&gt;) element.
     /// </remarks>
-    protected virtual void RenderDataCell (
```

Comment:
The method summary is no longer correct since the method is not `private` and `virtual`. We either remove it or move and adapt it to the new `virtual` method

Reply:
move it to the new virtual API and update to fit the new context.